### PR TITLE
Update insider-risk-management-cases.md

### PR DESCRIPTION
**Pull request by msbemba**
added more information about Investigators having the ability to Turn off pseudonymize

fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10359

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/insider-risk-management-cases.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ To add a contributor to a case:
 
 Risk investigators can take action on a case in one of several methods, depending on the severity of the case, the history of risk of the user, and the risk guidelines of your organization. In some situations, you may need to escalate a case to a user or data investigation to collaborate with other areas of your organization and to dive deeper into risk activities. Insider risk management is tightly integrated with other Microsoft Purview solutions to help you with end-to-end resolution management.
 
+Built with privacy by design, users are pseudonymized by default. However, IRM investigators can manage pseudonymize by turning it off for the case, if the option  is enabled at the tenant level.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-cases.md]:
```suggestion
Built with privacy by design, users are pseudonymized by default. However, IRM investigators can manage pseudonymize by turning it off for a case if the option is enabled at the tenant level.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by robmazz** [comment]
@msbemba - thanks, I'll discuss this addition with the PG and Marketing teams about if we want to highlight this here.

**Pull request closed without merging**